packages/@pollyjs/adapter-fetch/src/index.js

```diff
@@ -81,6 +83,16 @@ export default class FetchAdapter extends Adapter {
     return this.respond(pollyRequest, statusCode, headers, body);
   }
 
+  async onIdentifyRequest(pollyRequest) {
+    const { identifiers } = pollyRequest;
+    let body = identifiers.body;
+
+    body = await BrowserSerializers.serializeRequestBody(body);
+    body = NodeSerializers.serializeRequestBody(body);
+
+    identifiers.body = body;
```

Comment:
@jasonmit since both serializers are browser and node safe, should we just default to this logic on the core Adapter class and adapters that want to override this will have the ability to?

Reply:
>  override this

That seems fine.  Unrelated, I'm confused by the browser and node serializer being invoked here.

Reply:
After discussions on slack, we've decided to split the serializers by type (not environment) and run them all by default.